packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -211,6 +212,7 @@ export default async args => {
                       ...(config.babelConfig || {})
                   })
                 : null,
+            image(),
```

Comment:
This is necessary to directly import images in our code. But I'm not quite sure whether this will affect other widgets